### PR TITLE
DAOS-6837 test: datamover dst create

### PR DESCRIPTION
**Pull request by daltonbohning**
Test-tag: pr datamover

- Added test case to verify creation of destination containers.
- Added display_space to DataMoverTestBase.run_ior
- Added cont_type param to DataMoverTestBase.create_cont

### CODE REVIEW

File: src/tests/ftest/datamover/dm_dst_create.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        Use Cases:
+            Create pool1.
+            Create POSIX cont1 in pool1.
+            Create small dataset in cont1.
+            Copy cont1 to a new cont in pool1, with a supplied UUID.
+            Copy cont1 to a new cont in pool1, without a supplied UUID.
+            Create pool2.
+            Copy cont1 to a new cont in pool2, with a supplied UUID.
+            Copy cont1 to a new cont in pool2, without a supplied UUID.
+            For each copy, very container properties and user attributes.
```
Comment (shimizukko) [inline src/tests/ftest/datamover/dm_dst_create.py:55]:
```suggestion
            For each copy, verify container properties and user attributes.
```

**Review by shimizukko — APPROVED** [review]

**Review by dsikich — APPROVED** [review]

File: src/tests/ftest/datamover/dm_dst_create.py (hunk 1 above)
Comment (daltonbohning) [reply src/tests/ftest/datamover/dm_dst_create.py:55]: Unless there are other needed changes to the PR, I'll leave this spelling correction for a later patch. Thanks!

**Comment by sylviachanoiyee** [comment]
I'm not sure whether landing this now will cause the new testcase to fail on master due to a typo landed on master recently, currently being fixed in PR-5499
https://github.com/daos-stack/daos/pull/5499
is almost done testing. I think I will wait to land that first, then this one.

**Review by sylviachanoiyee — CHANGES_REQUESTED** [review]
Waiting to land the following first
https://github.com/daos-stack/daos/pull/5499

**Pull request merged**